src/main.cpp

```diff
@@ -38,6 +38,7 @@ constexpr int kParseCmdlineArgsErrorExitCode = 2;
 constexpr char kScaleFactorEnvVar[] = "QT_SCALE_FACTOR";
 const QString kConfigGroup = QStringLiteral("[Config]");
 const QString kScaleFactorKey = QStringLiteral("ScaleFactor");
+const QString kNotifyMaxDbgTimeKey = QStringLiteral("NotifyMaxDbgTime");
```

Comment:
Is this a new setting key? If yes, it would be best to use snake_case as per our coding guidelines 

```suggestion
const QString kNotifyMaxDbgTimeKey = QStringLiteral("notify_max_dbg_time");
```

Reply:
Oh, of course. Usually I just stick to the scheme that's used in the file, but then we'd not move forward obviously.
Thanks! Fixed, force-pushed.